src/healthkit.cs

```diff
@@ -3099,6 +3163,16 @@ interface HKUnit : NSCopying, NSSecureCoding {
 		[Static]
 		[Export ("degreeAngleUnit")]
 		HKUnit DegreeAngle { get; }
+
+		[Watch (10, 0), MacCatalyst (17, 0), Mac (14, 0), iOS (17, 0), NoTV]
+		[Static]
+		[Export ("luxUnitWithMetricPrefix:")]
+		HKUnit GetLuxUnit (HKMetricPrefix prefix);
+
+		[Watch (10, 0), MacCatalyst (17, 0), Mac (14, 0), iOS (17, 0), NoTV]
+		[Static]
+		[Export ("luxUnit")]
+		HKUnit GetLuxUnit ();
```

Comment:
We've bound other "*Unit" API without the "Unit" suffix in this type, and as a property:

```suggestion
		HKUnit Lux { get; }
```

this also matches the name in Swift.